x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/context_menu_actions.tsx

```diff
@@ -55,7 +74,7 @@ export const ContextMenuActions = React.memo<Props>(({ button, ...props }) => {
           />
         )
       }
-      isOpen={isOpen}
+      isOpen={typeof isOpen !== 'undefined' ? isOpen : isOpenState}
```

Comment:
prefer to avoid string typeof comparison, when possible:
 
```suggestion
      isOpen={isOpen === undefined ? isOpenState : isOpen}
```